src/test/scala/org/broadinstitute/dsde/workbench/sam/directory/LdapDirectoryDAOSpec.scala

```diff
@@ -48,6 +53,11 @@ class JndiDirectoryDAOSpec extends FlatSpec with Matchers with TestSupport with
       runAndWait(dao.createGroup(group))
     }
 
+    val conflict = intercept[WorkbenchExceptionWithErrorReport] {
```

Comment:
The name of the spec is `"JndiGroupDirectoryDAO" should "create...` should be renamed to `LdapGroupDirectoryDAO`

Reply:
fixed